### DISCUSSION TITLE
Version 2.2.1

### DISCUSSION THREAD

**Discussion by guydavis** [guydavis/machinaris#978] [Announcements]
Version 2.2.1 of Machinaris is now available, for those that like to try the very latest Chia release from CNI corporate.

### Updated
 - [Chia](https://github.com/Chia-Network/chia-blockchain/releases/tag/2.2.1) to v2.2.1 - possible fixes for serious problems in v2.2.0?
### Notes
 - Updated potentially buggy Chia software DOES NOT imply my endorsement.  *Only upgrade when you are comfortable.*

<hr /><em>This discussion was created from the release <a href='https://github.com/guydavis/machinaris/releases/tag/2.2.1'>Version 2.2.1</a>.</em>

**Comment by colbyboles** [comment]
Every time I install this as "latest" or even explicitly as v2.2.1, when I check the chia version it is 2.1.4.dev0?

**Reply by guydavis** [reply]
On Unraid? Try a "Force Upgrade" as that platform can often fail to find an updated image.  Hope that helps.
